Addon/Addon.h

```diff
@@ -12,41 +12,42 @@
 #include <string>
 #include <vector>
 
-#include "qwidget1.h"
-#include "SlicesHandlerInterface.h"
 #include "AddonApi.h"
+#include "SlicesHandlerInterface.h"
+#include "qwidget1.h"
 
-namespace iseg 
-{
-	class CSliceHandlerInterface;
+namespace iseg {
+class CSliceHandlerInterface;
 }
 
-namespace iseg { namespace plugin
+namespace iseg { namespace plugin {
+
+class ADDON_API CAddon
 {
+public:
+	CAddon();
+	~CAddon();
 
-	class ADDON_API CAddon
+	void SetSliceHandler(CSliceHandlerInterface *slice_handler)
 	{
-	public:
-		CAddon();
-		~CAddon();
-
-		void SetSliceHandler(CSliceHandlerInterface* slice_handler) { _slice_handler = slice_handler; }
-		CSliceHandlerInterface* SliceHandler() const { return _slice_handler; }
-
-		virtual std::string Name() const = 0;
+		_slice_handler = slice_handler;
+	}
+	CSliceHandlerInterface *SliceHandler() const { return _slice_handler; }
```

Comment:
why is this one a one liner and SetSliceHandler not? any reason?
also the indentation appears weird.